generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb

```diff
@@ -0,0 +1,908 @@
+{
```

Comment:
<p>We should add a little description of what the method does for you so one doesn't have to go to the paper for just the basics.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done!</p>